sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -526,7 +526,7 @@ def reader(self, test_bigquery_client=None):
 
 
 def _to_bool(value):
-  return value == 'true'
+  return value
```

Comment:
could you explain why this change is needed?

Reply:
the coder incorrectly expected boolean types to be encoded as strings. This is incorrect, as JSON supports boolean 'natively', like this: https://json-schema.org/understanding-json-schema/reference/boolean.html

Reply:
Do we need the _to_bool function then, would the bool casting suffice?

Reply:
ahhh what a great catch. thanks Yichi!